config/initializers/job_configurations.rb

```diff
@@ -199,6 +200,11 @@
         class: 'HeartbeatJob',
         cron: cron_5m,
       },
+      # Queue psql stats job to GoodJob
+      psql_stats_job: {
+        class: 'PsqlStatsJob',
+        cron: cron_1m,
+      },
```

Comment:
```suggestion
        cron: cron_1m,
        args: -> { [Time.zone.now] },
      },
```
need to pass this arg so we can use it for the concurrency key

Reply:
all set